cwltool/pathmapper.py

```diff
@@ -246,3 +248,9 @@ def reversemap(self, target):  # type: (Text) -> Tuple[Text, Text]
             if v[1] == target:
                 return (k, v[0])
         return None
+
+    # On windows os.path.join would use backslash to join path, since we would use these paths in Docker we would convert it to /
+    def pathFix(self,path):  # type: (Text) -> (Text)
```

Comment:
Why is this a class method? `self` is not used

Should have a different name than https://github.com/common-workflow-language/cwltool/pull/419/files#diff-29ce315cc581b507c2832e379ce71ad6R137